packages/material-ui/src/FormControl/FormControl.test.js

```diff
@@ -83,68 +94,89 @@ describe('<FormControl />', () => {
 
   describe('prop: disabled', () => {
     it('will be unfocused if it gets disabled', () => {
-      const wrapper = mount(<FormControl />);
-      setState(wrapper, { focused: true });
+      const testFunction = spy();
+      const wrapper = mount(
+        <FormControl>
+          <Input />
+          <TestComponent fn={testFunction} />
+        </FormControl>,
+      );
+      assert.strictEqual(testFunction.calledWithMatch({ focused: false }), true);
+      wrapper.find('input').simulate('focus');
+      assert.strictEqual(testFunction.calledWithMatch({ focused: true }), true);
```

Comment:
How does `calledWithMatch` work? From what I understand, spy is called twice here. Does it check the last call instance or any of the instances? (It's also why I used `.args()[x][y]`, there are no ambiguities possible).

Reply:
Yeah, it does allow for ambiguity. I'll change it :+1: